tools/database/overrides.go

```diff
@@ -97,6 +97,7 @@ var ItemOverrides = []*proto.UIItem{
 
 // Keep these sorted by item ID.
 var ItemAllowList = map[int32]struct{}{
+	19918:  {}, 
```

Comment:
Looks like a new version was added so this can be removed

Reply:
removed